### DISCUSSION TITLE
Can I calculate  the percentage of transcript models with different ranges of sequence coverage by long reads

### DISCUSSION THREAD

**Discussion by YIGUIz** [ConesaLab/SQANTI3#333] [Q&A] [answered]
Hi,

When I read the systematic assessment paper, I noticed that They calculate the percentage of transcript models with different ranges of sequence coverage by long reads. And I also want to calculate it. So Does SQANTI3 have this function? Or can I calculate with other software?

Thank you for your help.

Qi

**Comment by carolinamonzo** [comment, marked as answer]
Hi @YIGUIz, I'm not sure to which plot you are referring to. SQANTI3 is for quality control, so it mostly gives feedback on how reliable are the transcript models. For transcript model reconstruction and quantification, there are many tools discussed in the systematic assessment paper that calculate the coverage, for example FLAIR, IsoQuant, Bambu, Mandalorion etc.
